GameDataParser/Files/MetadataExporter/Exporter.cs

```diff
@@ -0,0 +1,20 @@
+namespace GameDataParser.Files
+{
+    public abstract class Exporter<Metadata> : MetadataExporter
+    {
+        protected MetadataResources resources;
+
+        public Exporter(MetadataResources resources, string slug) : base(slug)
+        {
+            this.resources = resources;
+        }
+
+        protected override void serialize()
```

Comment:
I notice that the new functions are all camel case, it's not necessary for this PR but I think we should decide if we want to change all functions to camel case or not

Reply:
I can change it, coming off of TypeScript